JavaFXUserInterface/src/main/java/Controllers/AssetTypeController.java

```diff
@@ -124,7 +129,10 @@ public void fillAssetTypeTable() {
     public void attachEvents() {
         //set up the columns in the table
         attachColumnEvents();
-
+        // Change scenes to Assets.fxml
+        backBtn.setOnMouseClicked(mouseEvent -> uiUtilities.changeScene(mouseEvent, TextConstants.ASSETS_SCENE));
```

Comment:
I don't think there should be a back button on the main screens of the side left menu. Asset types screen and assets screen is main screens on their own. A back button for coming back to the assets from the asset types is the same as clicking on the left-side menu.

Reply:
Also, you could have reached that screen from a specific asset info screen. However, clicking on the back button will always take you back to the assets screen.

Reply:
This issue was addressed